salt/_beacons/metalk8s_kubeconfig_info.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+from salt.ext.six.moves import map as _map
+
+__virtualname__ = 'metalk8s_kubeconfig_info'
+
+DEFAULT_NOTIFY_DAYS = 15
+
+log = logging.getLogger(__name__)
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def validate(config):
+    error_msg = "Configuration for {0} beacon is invalid".format(
+        __virtualname__
+    )
+
+    if not isinstance(config, list):
+        return False, "{0}: it must be a list".format(error_msg)
+
+    _config = {}
+    list(_map(_config.update, config))
+
+    if "files" not in config:
```

Comment:
That's a typo I think:

```suggestion
    if "files" not in _config:
```

Reply:
It'is, good catch.

Reply:
Fixed